plugin.go

```diff
@@ -559,3 +562,93 @@ func (p *ormPlugin) generateListHandler(message *generator.Descriptor) {
 	p.P(`}`)
 	p.P()
 }
+
+func (p *ormPlugin) generateDefaultServer(file *generator.FileDescriptor) {
+	for _, service := range file.GetService() {
+		svcName := lintName(generator.CamelCase(service.GetName()))
+		if service.Options != nil {
+			v, err := proto.GetExtension(service.GetOptions(), gorm.E_Server)
+			opts := v.(*gorm.AutoServerOptions)
+			if err == nil && opts != nil && *opts.Autogen {
+				// All the default server has is a db connection
+				p.P(`type `, svcName, `DefaultServer struct {`)
+				p.P(`DB `, p.gormPkgAlias, `.DB`)
+				p.P(`}`)
+				for _, method := range service.GetMethod() {
+					methodName := lintName(generator.CamelCase(method.GetName()))
+					if strings.HasPrefix(methodName, "Create") {
+						p.generateCreateServerMethod(file, service, method)
+					} else if strings.HasPrefix(methodName, "Read") {
+						p.generateReadServerMethod(file, service, method)
+					} else if strings.HasPrefix(methodName, "Update") {
+						p.generateUpdateServerMethod(file, service, method)
+					} else if strings.HasPrefix(methodName, "Delete") {
+						p.generateDeleteServerMethod(file, service, method)
+					} else if strings.HasPrefix(methodName, "List") {
```

Comment:
if possible, it would be nice to check if the `.proto` service return message actually has a `Results` field of the proper type and render something that compiles either way. This current approach means not fitting the convention results in code that doesn't compile. This would be nice to do for all of these methods.

Reply:
I agree and it seems that it's possible to implement that, though it's not trivial. Along with return type we need to validate method parameter as well, so I would like to come back to this when the positive cases are 100% working. This server is generated only if `(gorm.server).autogen` option is explicitly set, so the chance that the user knows what he is doing increases. 
In case of failed validation we may just leave `return nil, nil` or `return nil, &NotImplementedError{"MethodName"} as a default implementation, that will always compile.